### DISCUSSION TITLE
How to use MediaMTX's multicast

### DISCUSSION THREAD

**Discussion by Double-Lin** [bluenviron/mediamtx#3912] [Questions]
### Question

How to use MediaMTX's multicast:
What instruction does ffmpeg use to push to mediamtx?
2. Which address does the player access to play multicast videos?
thank you

**Comment by aler9** [comment]
If you want to read streams with the multicast variant of RTSP, and the reader is VLC, instructions are here:
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#vlc
